qemu/tests/cdrom.py

```diff
@@ -398,7 +362,8 @@ def get_testing_cdrom_device(session, cdrom_dev_list, serial_num=None):
                 cdrom_dev_list.remove(winutil_drive)
                 testing_cdrom_device = cdrom_dev_list[-1]
             else:
-                testing_cdrom_device = get_match_cdrom(session, serial_num)
+                testing_cdrom_device = utils_misc.get_linux_drive_path(
+                    session, serial_num)
```

Comment:
For the 2nd parameter, it can be a wwn, which requires int type.
but the parameter which is transferred is a str.